typescript-timers/interval.ts

```diff
@@ -0,0 +1,14 @@
+function change(): void {
+  const $countdown = document.querySelector('.countdown-display');
```

Comment:
`querySelector`s should usually be done globally. Especially in functions that are in a `setInterval`. Query selectors can be very inefficient. By including it inside the function, you're forcing us to traverse the DOM every single time this function gets called